vignettes/themeBuilder_addin.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "Theme Configuration Builder"
+author: "Mohammed Ali"
+date: "`r Sys.Date()`"
+output: 
+    rmarkdown::html_vignette:
+        toc: TRUE
+        toc_depth: 3
+vignette: >
+  %\VignetteIndexEntry{Theme Configuration Builder}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Overview
+
+To help package users in creating valid theme configuration file
+
+## Add-in Launch
+
+The add-in can be lunched by one of two methods
```

Comment:
Spelling: "lunched" should be "launched"

Reply:
done